### ISSUE TITLE
Import weirdness

### ISSUE THREAD

**Issue by ryanmrichard** [NWChemEx-Project/.github#34]
Admittedly I'm pretty weak when it comes to the mechanics of Python's import mechanism, so this might be a Python error on my side.

Anyways, since this works:
```.py
import simde

mol = simde.chemist.Molecule()
```

I would have expected:
```.py
import simde.chemist as chemist

mol = chemist.Molecule()
```

to work. Instead I get `ModuleNotFoundError: No module named 'simde.chemist'`. Assuming this isn't a problem with my Python, I think what's happening is that the Chemist found by SimDE isn't actually a Python module (i.e., it doesn't have an `__init__.py` file). Interestingly:

```.py
import simde
import chemist

mol = chemist.chemist.Molecule()
```
works which suggests to me that we might have a Python stack where each module x includes copies of the repos below it, instead of simply than importing them as submodules.

I'm going to throw this issue in the .github repo, but it should be moved to the appropriate repo if .github isn't it.

**Comment by wadejong** [comment]
This is a standard Python feature. The solution that we need to use is "from x import y". Hence:

```
from chemist import chemist     or      from simde import chemist    or from nwchemex import chemist

mol = chemist.Molecule() 
```

**Comment by ryanmrichard** [comment]
I don't think this is the same as #31. I think this is a problem with us actually copying the subpackages instead of importing them.
